modules/@apostrophecms/page/index.js

```diff
@@ -1867,7 +1867,12 @@ database.`);
           } else {
             parentSlug = item.parent;
           }
-          return self.findOneForEditing(req, { slug: parentSlug });
+          return self.findOneForEditing(req, {
+            slug: parentSlug
+          }, {
+            areas: false,
```

Comment:
We don't need all the relationships of the home page or other parked page parent to park its children. This makes parked pages fast again.